### DISCUSSION TITLE
Redirect with wildcard in subdomain

### DISCUSSION THREAD

**Discussion by kwschnei** [mshibanami/redirect-web#13] [Questions] [answered]
I'm attempting to write a redirect rule because I often expect autocomplete to finish URLs for me and it has learned a couple bad examples.

I'd like to redirect `https://*.foo.com/ad` to `https://*.foo.com/admin` - where it remembers the URL that it matches in the Redirect From and uses that in the Redirect To.

Examples:
https://bar.foo.com/ad redirects to https://bar.foo.com/admin
AND https://bob.foo.com/ad redirects to https://bob.foo.com/admin

Is this possible?

**Comment by mshibanami** [comment, marked as answer]
Hi @kwschnei, yes, it's possible. You can use `$1` to back-reference the string captured by `*`.
This is a feature of regex, but since this extension internally converts your wildcard pattern to a regex pattern and `*` and `?` are automatically captured, you can use $1, $2, ... with a wildcard pattern.

You can find the details here: https://mshibanami.github.io/redirect-web/#/redirect-rule?id=wildcard

<img width="400" alt="Screenshot 2023-09-01 at 12 53 34 pm" src="https://github.com/mshibanami/redirect-web/assets/1333214/72abff4e-e3e9-4b7d-b168-70074297774a">

**Comment by kwschnei** [comment]
I spent a bunch of time looking through the wiki and I'm not sure how I missed that. Thanks so much for your patience in answering a question that you already had!
